src/Uno.UI/UI/Xaml/Media/Animation/ColorAnimation.cs

```diff
@@ -116,14 +116,9 @@ public bool EnableDependentAnimation
 		/// <summary>
 		/// Dispose the Double animation.
 		/// </summary>
-		protected override void Dispose(bool disposing)
+		~ColorAnimation()
 		{
-			if (disposing)
-			{
-				_animationImplementation.Dispose();
-			}
-
-			base.Dispose(disposing);
+			_animationImplementation.Dispose();
 		}
```

Comment:
This may be incorrect, as it may call UI Thread-affine APIs on the GC's thread. It may be needed to schedule the disposable on the dispatcher, somehow.

Reply:
I was expecting that I might be doing something wrong, but I couldn't really figure out a different way.

> It may be needed to schedule the disposable on the dispatcher, somehow.

Just a `_ = Dispatcher.RunAsync(...)`?